spec/features/contact_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe 'Contact Page' do
+
+  before do
+    visit root_path
+  end
+
+  context "when there are no settings configured" do
+    context "and the contact page is visited" do
+      before do
+        click_link "Contact Us"
+      end
+      it 'should say there are no settings configured' do 
+        expect(page).to have_content("No settings are configured.")
+      end
+    end
+  end
+
+  context "when there are settings configured" do
+    let(:setting){create(:setting, :setting_name => 'Contact', :value => "Yo!")}
+    before do
+      setting
+    end
+    context "and the contact page is visited" do
+      before do
+        click_link "Contact Us"
+      end
+      it 'should say there are no settings configured' do 
```

Comment:
Definitely! Ill get that fixed asap.
On Feb 3, 2015 12:45 PM, "Trey Terrell" <notifications@github.com> wrote:

> In spec/features/contact_spec.rb
> <https://github.com/osulp/building-oregon/pull/26#discussion_r24038056>:
>
> > +      it 'should say there are no settings configured' do
> > +        expect(page).to have_content("No settings are configured.")
> > +      end
> > +    end
> > +  end
> > +
> > +  context "when there are settings configured" do
> > +    let(:setting){create(:setting, :setting_name => 'Contact', :value => "Yo!")}
> > +    before do
> > +      setting
> > +    end
> > +    context "and the contact page is visited" do
> > +      before do
> > +        click_link "Contact Us"
> > +      end
> > +      it 'should say there are no settings configured' do
>
> @straleyb <https://github.com/straleyb> Can you look at this note?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/osulp/building-oregon/pull/26/files#r24038056>.
>